lib/caeser.rb

```diff
@@ -0,0 +1,67 @@
+require 'pry'
+class Caeser
+
+  def initialize(shift_value)
```

Comment:
Read the spec more carefully. A Caesar object does not take any parameters. The shift_value is passed to the eng_to_cipher method